docs/standards.md

```diff
@@ -22,17 +22,37 @@ Additionally, here are some common variable suffixes that are recommended:
 `_f`    for a pointer to a function
 `_ctx`  for context data i.e. data that is passed into a function
 
-### Functions
+#### Functions
 Exported function names are prefixed, and lowercase snake case.
+
+The prefix used is based on the name of the dynamic library and the functionality it provides.
+```
+  DynamicLibraryLoaderHelper -> DLLH_
+  Memory Allocation -> Mem_
+  MicrophoneUtility -> MicrophoneUtility_
+```
+
+Prefacing the definition of the function with `FUN_EXPORT` is doen to allow for the code to be defined and exported under
+different calling convetions and different linking types (static and dynamic).
+
 e.g.
 `FUN_EXPORT(bool) DLLH_unload_library_at_path(void *ctx, void *library_handle)`
 
-The Epic Online Services SDK has it's one standards and style [guide](https://docs.unrealengine.com/4.26/en-US/ProductionPipelines/DevelopmentSetup/CodingStandard/).
+For reference, the Epic Online Services SDK has it's one standards and style [guide](https://docs.unrealengine.com/4.26/en-US/ProductionPipelines/DevelopmentSetup/CodingStandard/).
 
 ### For C#
+
+#### Variables
+* static variables should start with `s_`, globals with `g_`.
```

Comment:
Unless they are constants, in which case they should be `ThisCase` (can't remember what that's called)

Reply:
also should use the ticks to make it `static` instead of static